### ISSUE TITLE
After Load event

### ISSUE THREAD

**Issue by alexandrekumagae** [Rplus/lazy-blur.js#3]
It would be nice if there were an afterLoad event on the plugin..

**Comment by Rplus** [comment]
@alexandrekumagae , did you mean the `onLoad` parameter in options?
https://github.com/Rplus/lazy-blur.js#options

**Comment by alexandrekumagae** [comment]
No, I want to start some animations and show some content only when finished loading the image

**Comment by Rplus** [comment]
Hmmm, I think `onLoad` is what you need.
It'll fire after the big image is loaded each time.

Or, is what you want 'a event for all images are loaded'?
